documentation/configuration.md

```diff
@@ -21,5 +21,19 @@ metrics: {
             enabled: true
         }
     }
+
+    jolokia {
+      # Enable or disable the Jolokia-based metrics/v2 endpoint.
+      # Default is true.
+      enabled: false
+
+      # Configure any of the settings listed at:
+      #   https://jolokia.org/reference/html/agents.html#war-agent-installation
+      config: {
+        # Specify a custom security policy:
+        #  https://jolokia.org/reference/html/security.html
+        policyLocation: "file:///etc/puppetlabs/<service name>/jolokia-access.xml"
```

Comment:
I'm okay sticking with the name `config`.  It just seemed a little redundant to me at first since the whole section is "configuration" anyway.  But you raise a good point about the servlet being implementation detail and I'm not coming up with any better names.